packages/babel-template/src/populate.js

```diff
@@ -15,7 +15,10 @@ export default function populatePlaceholders(
       if (
         !Object.prototype.hasOwnProperty.call(replacements, placeholder.name)
       ) {
-        throw new Error(`No substitution given for "${placeholder.name}"`);
+        throw new Error(`Error: No substitution given for "NODE_ENV". If this is not meant to be a
+          placeholder you may want to consider passing one of the following options to babel-template:
+          - { placeholderPattern: false, placeholderWhitelist: new Set(['NODE']) }
```

Comment:
Not sure if 
`new Set(['NODE']`

should be this ?
`new Set(['NODE_ENV']`

Reply:
The error message should continue using `placeholder.name`, don't think we want to say `NODE_ENV` for every error :)

Reply:
@existentialism 
Ah... of course!